src/Dependencies.props

```diff
@@ -1,5 +1,5 @@
 <Project>
   <PropertyGroup Label="Framework Versions">
-    <ResourceProjectTargetFrameworks>netstandard2.0;netcoreapp3.1;net6.0;net7.0</ResourceProjectTargetFrameworks>
```

Comment:
I don't think we can drop support for netstandard2.0 as it would no longer support any dotnet framework version (and some of them are not EOL yet, see: https://learn.microsoft.com/en-us/lifecycle/products/microsoft-net-framework)